drivers/tee/optee/optee_private.h

```diff
@@ -60,6 +60,18 @@ static inline void optee_suppl_rpmb_release(struct udevice *dev)
 }
 #endif
 
+#ifdef CONFIG_DM_I2C
```

Comment:
might be better to move the ifdef to the code in i2c. This may error on build if CONFIG_DM_I2C is not set.

Reply:
um, how? what would be the undefined?

Reply:
Not undefined but multiply defined.  that is:
tools/gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf/bin/arm-none-linux-gnueabihf-ld.bfd: drivers/tee/optee/supplicant.o: in function `optee_suppl_cmd_i2c_transfer':
u-boot/drivers/tee/optee/optee_private.h:71: multiple definition of `optee_suppl_cmd_i2c_transfer'; drivers/tee/optee/core.o:
u-boot/drivers/tee/optee/optee_private.h:71: first defined here
tools/gcc-arm-9.2-2019.12-x86_64-arm-none-linux-gnueabihf/bin/arm-none-linux-gnueabihf-ld.bfd: drivers/tee/optee/rpmb.o: in function `optee_suppl_cmd_i2c_transfer':
u-boot/drivers/tee/optee/optee_private.h:71: multiple definition of `optee_suppl_cmd_i2c_transfer'; drivers/tee/optee/core.o:
u-boot/drivers/tee/optee/optee_private.h:71: first defined here
u-boot/scripts/Makefile.build:355: recipe for target 'drivers/tee/optee/built-in.o' failed
make[4]: *** [drivers/tee/optee/built-in.o] Error 1

Reply:
um, but how is this possible? I dont get it...I mean the file i2c.c doesnt get build unless CONFIG_DM_I2C is defined so how can there be multiple definitions of the function?